docs/samples/Microsoft.ML.AutoML.Samples/Microsoft.ML.AutoML.Samples.csproj

```diff
@@ -0,0 +1,46 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\src\Microsoft.ML.Auto\Microsoft.ML.Auto.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Data\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="..\..\..\test\data\optdigits-test.csv" Link="Data\optdigits-test.csv">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="..\..\..\test\data\optdigits-train.csv" Link="Data\optdigits-train.csv">
```

Comment:
Can we reference the dataset thru the Dataset Utils? Given its part of examples, I expect it's accessible thru nugets. 

We could add the optdigits to SamplesDatasetUtils.cs:
https://github.com/dotnet/machinelearning/blob/fde1ab718bcf0da10293e5ca32245373e4669e2a/src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs#L18-L24

Reply:
Yeah, the opt-digits dataset is part of ML.NET samples. However, DatasetUtils doesn't download the dataset from NuGet, but instead pulls it from the ML.NET samples Git repository. You're right -- since this dataset is already in one Git repository (the ML.NET samples Git repo), no need to add it to another Git repository (the ML.NET one), too. Of course, questions about inconsistency come to mind (like, why have 2 out of 3 datasets for these samples (including two 25 MB ones), checked in directly, and download the other dataset (< 1MB) from another Git repo owned by our team), but I'm okay with that. Long-term, it'd be great to nail down a solid story around how we store datasets ofc. (Are datasets housed in their own repo? In Git LFS?  Etc)

Reply:
Correct. I was wondering if the DatasetUtils class is public in the nuget. The dataset files are downloaded via https. 

Longer term, I'm a fan of Git LFS, but we haven't set it up for the repo. There's an ok argument to keep all datasets in the samples repo too. Besides for unit test datasets which tend to be trivially small, I like having the main repo be rather slim by keeping datasets elsewhere. 

The DatasetUtils method also lets us reference larger datasets for our examples.